packages/components/src/lozenge/docs/Lozenge.stories.mdx

```diff
@@ -12,18 +12,35 @@ import { Text } from "@components/typography";
 
 # Lozenge
 
-<Tagline>Lozenges are used to highlight an item's status for quick recognition or to give additional fast short information such as highlighting a new item.</Tagline>
+<Tagline>Lozenges are used to highlight an item's status, for quick recognition or to give additional information such as highlighting a new item.</Tagline>
 
 <ComponentInfo
     usage={"import { Lozenge } from \"@sharegate/orbit-ui\";"}
     githubPath="/packages/components/src/lozenge/src"
 />
 
+## Best Practices
+
+- Use to highlight an element in a list.
```

Comment:
I'm wondering if these should be listed under "Usage" bellow as a list of scenarios we can use the Lozenge for.

Reply:
Usage is used in a how to use the component in code, that's why best practices wouldn't fit under usage section.